toolkit/tools/pkg/imagecustomizerlib/imagecustomizer.go

```diff
@@ -806,6 +813,57 @@ func validateOutput(baseConfigPath string, output imagecustomizerapi.Output, out
 	return nil
 }
 
+func validateStorage(storage *imagecustomizerapi.Storage) error {
+	if storage == nil {
+		return nil
+	}
+
+	err := validateVeritySignatureInjection(storage.Verity, storage.FileSystems)
+	if err != nil {
+		return err
+	}
+
+	return nil
+}
+
+func validateVeritySignatureInjection(verityList []imagecustomizerapi.Verity,
+	fileSystems []imagecustomizerapi.FileSystem,
+) error {
+	if len(verityList) == 0 {
+		return nil
+	}
+
+	var espMount string
+
+	for _, fs := range fileSystems {
```

Comment:
This isn't how you would find the ESP. Instead, you should use partition type UUID. `Storage.IsValid` function already has this logic.